contracts/cosmwasm-vm/README.md

```diff
@@ -33,10 +30,26 @@ rustup target add wasm32-unknown-unknown
 ```
 
 For windows,
-Download and run, [rustup-init.exe](https://static.rust-lang.org/rustup/dist/i686-pc-windows-gnu/rustup-init.exe)
+Download and run, [rustup-init.exe](https://static.rust-lang.org/rustup/dist/i686-pc-windows-gnu/rustup-init.exe)  
+
+You can also execute the `./scripts/optimize-cosmwasm.sh`  script, which handles installing the necessary toolchain dependencies, building, and optimizing the contracts to their correct versions.
 
 ### Build Contracts
 
+1. Using docker build  
+```
+make optimize-cosmwasm    # The final artifacts will reside within the `./artifacts/archway` directory.
+```
```

Comment:
```suggestion
**1. Docker Build Process**

To initiate the build process using Docker, it is essential to have the "contract-builder" image. To generate this Docker image, execute the subsequent command:

```
make build-builder-img
```

In case you possess the "contract-builder" image already, you can directly employ the following command:

```
make optimize-cosmwasm    # The resulting artifacts will be located within the `./artifacts/archway` directory.
```
```